doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -651,11 +656,13 @@ Libraries for networking
     * An :c:struct:`nrf_cloud_obj_shadow_data` structure to the :c:struct:`nrf_cloud_evt` structure to be used during shadow update events.
     * The :kconfig:option:`CONFIG_NRF_CLOUD_SEND_SERVICE_INFO_FOTA` Kconfig option to enable sending configured FOTA service info on the device's initial connection to nRF Cloud.
     * The :kconfig:option:`CONFIG_NRF_CLOUD_SEND_SERVICE_INFO_UI` Kconfig option to enable sending configured UI service info on the device's initial connection to nRF Cloud.
+    * An :c:struct:`nrf_cloud_location_config` structure for specifying the desired behavior of an nRF Cloud ground-fix request.
```

Comment:
We should mention somewhere that `bool request_loc` parameter has been converted to the new structure. This is important because it's backwards incompatible change.

Reply:
Added to the line that describes updates to the two functions that used to take `bool request_loc`:

>     * The :c:func:`nrf_cloud_obj_location_request_create` and :c:func:`nrf_cloud_location_request` functions to accept the :c:struct:`nrf_cloud_location_config` structure in place of the ``bool request_loc`` parameter.